packages/modeling/src/operations/hulls/hullPoints2.js

```diff
@@ -1,108 +1,66 @@
 const vec2 = require('../../maths/vec2')
 
-const angleBetweenPoints = (p0, p1) => Math.atan2((p1[1] - p0[1]), (p1[0] - p0[0]))
+// interface PolarPoint {
```

Comment:
Not sure what this comment adds. It doesn't explain anything about polar points or the contents.

Reply:
That was mostly a note-to-self about what was in the `points` objects. I removed the commented code now. Good catch.